Makefile

```diff
@@ -54,6 +66,35 @@ build-image-ocp-example: build-base-image ## Build a rag-content container image
 build-base-image: ## Build base container image
 	podman build -t $(TORCH_GROUP)-road-core-base -f Containerfile.base --build-arg FLAVOR=$(TORCH_GROUP)
 
+start-postgres:
```

Comment:
Target needs a help line. At least a short one, otherwise it will be ignored by the script. Or the script could be rewritten to print targets without help.

Reply:
Added a help line to each of new targets. Thanks.